scylla-server/src/index.ts

```diff
@@ -56,9 +56,12 @@ const serverSocket = new Server(server, {
   }
 });
 
+const proxyServers: ProxyServer[] = [];
```

Comment:
can I get some context on what these proxyServers are for?

Reply:
Theyre all the connections to the server. So if u connect and then i connect both connections will have their owjnprody server